File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    result = []
    answer = s.split(' ')
    
    for char in answer:
        if char != '':
            result.append(char[0].upper() + char[1:].lower())
        else:
            result.append(char)
        
    return " ".join(result)

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    let arr = [];
    let answer = s.split(' ');

    for (let i = 0; i < answer.length; i++) {
        if (answer[i]) {  
            arr.push(answer[i][0].toUpperCase() + answer[i].slice(1).toLowerCase());
        } else {
            arr.push(''); 
        }
    }
    
    return arr.join(' ');
}
